pylint/test/input/func_e0303.py

```diff
@@ -0,0 +1,44 @@
+"""Check bad value returned by __len__ """
```

Comment:
They are discovered through test_func.py indeed. What issues did you have? Anyway, please write them in the new format, these being executed through test_functional.py.